lib/stylesheet/importer.rb

```diff
@@ -64,9 +64,32 @@ def font
 
         CSS
 
+      # conditionally enable Jetbrains Mono font on rich editor
+      if SiteSetting.rich_editor
+        contents << <<~CSS
+          #{font_css(jetbrains_mono)}
+          #{render_font_special_properties(jetbrains_mono, "body")}
+          :root {
+            --d-font-family--monospace: #{jetbrains_mono[:stack]};
```

Comment:
I guess my only question is, do we care that this also affects code blocks and so on? If we do, might be better to define a `--d-font-family--editor-monospace` font to better separate things and then use that on `.d-editor-input`.

Reply:
I think it is important to hit both, cause is forces a consistency in the app, all code blocks everywhere look better.